xcodeproj/internal/bazel_integration_files/calculate_output_groups.py

```diff
@@ -249,10 +257,29 @@ def _similar_platforms(platform):
         return _SIMULATOR_PLATFORMS
     return _DEVICE_PLATFORMS
 
-def _main(action, objroot, base_objroot, scheme_target_id_file, prefixes_str):
+def _main(
+        action,
+        xcode_version,
+        objroot,
+        base_objroot,
+        scheme_target_id_file,
+        prefixes_str):
     if not os.path.exists(scheme_target_id_file):
         return
 
+    try:
+        xcode_version = int(xcode_version)
+    except ValueError:
+        print(
+            """
+warning: XCODE_VERSION_ACTUAL was not an integer. Please file a bug report \
```

Comment:
nit: Maybe output `xcode_version` so its easier to determine what it was set to by mistake.